pkg/input/keys.go

```diff
@@ -85,3 +86,74 @@ func GetVirusTotalKey() string {
 
 	return keys.VirusTotal
 }
+
+// GetBuiltWithKey reads the BuiltWith key.
+func GetBuiltWithKey() string {
+	filename := ""
+	if runtime.GOOS == windows {
+		filename = "keys.yaml"
+	} else { // linux
+		home, err := os.UserHomeDir()
+		if err != nil {
+			fmt.Println("Cannot read BuiltWith Api Key.")
+			os.Exit(1)
+		}
+		filename = home + "/.config/scilla/keys.yaml"
+	}
+
+	keys, err := ReadKeys(filename)
+	if keys.BuiltWith == "" {
+		fmt.Println("BuiltWith Api Key is empty.")
+		os.Exit(1)
+	}
+
+	if err != nil {
+		fmt.Println("Cannot read BuiltWith Api Key.")
+		os.Exit(1)
+	}
+
+	return keys.BuiltWith
+}
+
```

Comment:
I have the `GetBuiltWithKey()` ready for use, but since it shares so much base functionality with `GetVirusTotalKey()`, I created a `GetKey(api string)` function below that just returns the needed key based on an api string passed in. 

Usage is on lines 369 and 375 of `pkg/runner/runner.go`. If this is desirable, we can go back to the separate functions, but thought I'd attempt to dry things up here, lmk your thoughts.